README.md

```diff
@@ -73,8 +73,8 @@ crate-type = ["cdylib"]
 [trav-url]: https://travis-ci.org/rust-dsp/rust-vst
 [appv-img]: https://ci.appveyor.com/api/projects/status/npiyjfithlx50hfs?svg=true
 [appv-url]: https://ci.appveyor.com/project/rustdsp/rust-vst
-[crates-img]: https://img.shields.io/crates/v/vst2.svg
-[crates-url]: https://crates.io/crates/vst2
+[crates-img]: https://img.shields.io/crates/v/vst.svg
```

Comment:
I don't believe we're currently using a few images but names were changed just in case